packages/apps/web-viewer-test/src/App.tsx

```diff
@@ -10,11 +10,13 @@ import { AllRoutes } from "./components/routing";
 
 function App() {
   return (
-    <React.Suspense fallback={<></>}>
-      <BrowserRouter>
-        <AllRoutes />
-      </BrowserRouter>
-    </React.Suspense>
+    <React.StrictMode>
```

Comment:
wrap `<App />` with strict mode in index.tsx

Reply:
moved to wrap <App />